### PR TITLE
Improved Pipeline Automation documentation

### PR DESCRIPTION
**Pull request by victorskl**
* Pull up high level Q/A notes on what automation do with
  topup and rerun library, PTC/NTC samples, Lab Metadata
  new sheet procedure, ctTSO run dir dependency caveat.

### CODE REVIEW

File: docs/pipeline/automation/transcriptome.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
...
+## Notes
+
+- See [metadata.md](../metadata.md) note for NTC/PTC sample treatment.
+- Since Portal 2.2 (Ocicat) release, DRAGEN transcriptome trigger business logic has updated as follows.
+  - For **topup** library, it will trigger "merged" with the original (initially) sequenced WTS library.
+  - For **rerun** library, it will skip trigger all together. Hence, it requires manual trigger follow up; as well as for the corresponding RNAsum.
+  - DRAGEN transcriptome workflow is triggered in Subject-wide manner. 
+    - i.e. It could go across different sequencing runs to find matching WTS library (e.g. topup) of a given subject.
+
+
```
Comment (victorskl) [inline docs/pipeline/automation/transcriptome.md:18]: @skanwal Trascriptome trigger behaviour has changed, significantly with Ocicat. So, added notes here. This is from yesterday discussion. fyi

File: docs/pipeline/automation/tumor_normal.md [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+## Notes
+
+- See [metadata.md](../metadata.md) note for NTC/PTC sample treatment.
+- In most recent cases, you should be able to drive Tumor/Normal trigger through Portal Launch Pad UI.
+- The following options are alternates; when you need to "pair" sample manually and, like to trigger them - [ditto](https://umccr.slack.com/archives/CP356DDCH/p1698793560313359).
+- Pairing algorithm
+  - Topup (library having `_topup`) are merged with previous libraries of the same name
+  - No logic yet for reruns (i.e. it will skip library having `_rerun` suffix)
+  - https://github.com/umccr/data-portal-apis/pull/262
+  - https://github.com/umccr/data-portal-apis/pull/596
```
Comment (victorskl) [inline docs/pipeline/automation/tumor_normal.md:15]: Hi @ohofmann (and @alexiswl) Pls feel free to review this T/N notes; and below and above. Have a comment, if any. Just documenting current status quo. fyi

File: docs/pipeline/metadata.md [hunk 3]
```diff
@@ -75,11 +97,15 @@ This activity typically happens at the end of year or, start of the year.
...
+    - https://github.com/umccr/data-portal-apis/pull/535
+    - https://github.com/umccr/data-portal-apis/pull/653
 3. Create Infrastructure PR to update SubjectID generator App script
     - https://github.com/umccr/infrastructure/blob/master/scripts/umccr_pipeline/UMCCR_Library_Tracking_MetaData.js#L5
     - https://github.com/umccr/infrastructure/pull/344
+    - https://github.com/umccr/infrastructure/pull/391
 4. Deploy SubjectID generator App script
+5. Update SampleSheet Checker UI (if any)
+   - https://github.com/umccr/samplesheet-check-frontend/pull/40
+   - https://umccr.slack.com/archives/C94CQMNVA/p1704844116202929
```
Comment (victorskl) [inline docs/pipeline/metadata.md:109]: @williamputraintan Added sschecker here too. And future of Metadata Manager; keep an eye on  below trello card too; we shall drive from there. fyi

File: docs/pipeline/metadata.md [hunk 4]
```diff
@@ -66,6 +66,28 @@ Assay
...
+	- We do not explicitly filter NTC/PTC yet (rule wise)
+	- However, for transcriptome workflow, we only consider - `Phenotype`: `tumor`
+		- So, all NTC get filtered (inheritance)
+	- PTC are still getting processed
+		- If, only if, LabMetadata `Workflow` column must be either of: `clinical` or `research`
+
+- For ctTSO,
+	- We explicitly allow process NTC (https://github.com/umccr/data-portal-apis/issues/561)
+	- PTC are getting processed
+
```
Comment (victorskl) [inline docs/pipeline/metadata.md:90]: @mhlunimelb hey Helen - documented ^^ it now. :) fyi

**Review by reisingerf — APPROVED** [review]

**Comment by victorskl** [comment]
Merging...

**Pull request merged**